tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -230,6 +230,14 @@ private void includeCss(HttpServletRequest request, PrintWriter out) {
      */
     public static void includeJavaScript(HttpServletRequest request, Writer out)
             throws IOException, JSONException {
+        // load any FE debug config
+        final String feDebug = System.getProperty("CLDR_FE_DEBUG", "{}");
+        if (!feDebug.trim().isEmpty()) {
+            out.write("<!-- CLDR_FE_DEBUG property -->\n");
+            out.write("<script>cldrFeDebug=" + feDebug + ";</script>\n\n");
+        } else {
+            out.write("<script>cldrFeDebug={}; // CLDR_FE_DEBUG unset</script>\n\n");
```

Comment:
I wish we could avoid embedding js inside html inside java like this

Reply:
I wish we would delete SurveyTool.java and make it a static html file. We could probably get webpack to assemble this into the bundle.  I did it this way to make it easier to understand and to use. if unset, there's no other effect.

Reply:
there's no checking whether feDebug is a well-formed json string -- that seems like a drawback to storing it in a properties file, along with having to squeeze it onto a single line

Reply:
It's not meant to be JSON, it's arbitrary js code, which is shorter to write (see example). That's a lot harder to validate.